### ISSUE TITLE
job finished alert 

### ISSUE THREAD

**Issue by Deltaplc** [grbl/grbl#1822]
Good morning
Is it possible to add a buzzer to signal the end of gcode file execution?
Thank you
Hello
David

**Comment by breiler** [comment]
No, there is no concept of a gcode file in GRBL just a stream of commands. If you want signalling like this you need to do it from the sender.
